torchaudio/csrc/lfilter.cpp

```diff
@@ -80,7 +80,7 @@ void lfilter_core_generic_loop(
   }
 }
 
-torch::Tensor lfilter_core(
+std::vector<torch::Tensor> lfilter_core(
```

Comment:
semi-nit: using a std::tuple<at::Tensor, at::Tensor> should be faster and safer (type checking).